pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -192,3 +186,16 @@ func (h *netlinkHandle) GetLocalAddresses(dev, filterDev string) (sets.String, e
 	}
 	return res, nil
 }
+
+func (h *netlinkHandle) getLinkIndex(dev string) (int, error) {
```

Comment:
We need more unit tests with these kinds of refactoring.

Reply:
ok, i'll add it later

Reply:
ok，i‘ll add it later

Reply:
hi，ut done